.gitignore

```diff
@@ -300,6 +300,10 @@ ompi/mpiext/pcollreq/mpif-h/profile/pscan_init_f.c
 ompi/mpiext/pcollreq/mpif-h/profile/pscatter_init_f.c
 ompi/mpiext/pcollreq/mpif-h/profile/pscatterv_init_f.c
 
+ompi/mpiext/shortfloat/c/mpiext_shortfloat_c.h
```

Comment:
Minor typo in the "mpiext/shortfloat: Add 'shortfloat' MPI extension" commit message: "thier" -> "their" (I'm adding this comment here in the .gitignore because it's the first comment-able line in the specific commit in question).